client-src/elements/form-field-specs.js

```diff
@@ -310,7 +310,7 @@ export const ALL_FIELDS = {
     help_text: html`
         Explain why the web needs this change. It may be useful
         to describe what web developers are forced to do without
-        it. When possible, add links to your explainer
+        it. When possible, add links to your explainer (under Explainer link(s))
```

Comment:
Is there any way this text can link to the explainer_links section instead of just naming it?

Reply:
I believe so.

Reply:
But it doesn't help much since it doesn't seem to autoscroll.  Maybe something else on the page is overriding the default autoscroll to the anchor.